### DISCUSSION TITLE
Nix package

### DISCUSSION THREAD

**Discussion by mariuskimmina** [mondoohq/cnquery#1586] [General]
Hello,

I found that there was currently no package for cnquery on https://search.nixos.org/packages. Since the effort for maintaining such a package doesn't seem terribly high I went ahead and created one, which is currently an open PR: https://github.com/NixOS/nixpkgs/pull/251554

If this is not welcome I would of course remove it again.

**Comment by chris-rock** [comment]
Happy to support you on that one. Great idea

**Comment by mariuskimmina** [comment]
Update on this:

The PR has been merged, the nixpkg for cnquery can be found here:
https://search.nixos.org/packages?channel=unstable&show=cnquery&from=0&size=50&sort=relevance&type=packages&query=cnquery

It's currently on the unstable channel and should move to stable with the next release in May this year.
I found that someone else had also already created a package for cnspec 
https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=cnspec

This package is already on a stable channel but it's still showing our previous license. 
I made a PR to change the license and to add myself as a maintainer for this package as well, that has been merged and will be corrected with their next release. 

Do we want to add a mention of this in our docs? - maybe that should that wait until both packages are on a stable channel with the correct license?
